console/src/assets/i18n/en.json

```diff
@@ -743,6 +743,10 @@
     "TIMEOUT": "Timeout",
     "TIMEOUTINSEC": "Timeout in seconds",
     "ALLOWEDTOFAIL": "Allowed To Fail",
+    "ALLOWEDTOFAILWARN": {
+      "TITLE": "Warning",
+      "DESCRIPTION": "Be aware that by disabling an action to fail, the process to log in or sign in users of the current organization can be interrupted and access to the console to disable an action can be declined. We recommend setting up admin users in a seperate organization or trying scripts in a dev environment or a dev organization first."
```

Comment:
Whats the difference between login and sign in? I think probably all of the text will have that duplicated now

Reply:
its the same.. I'll remove the second occurrence and change all other languages

Reply:
done